org.nomacs.ImageLounge.json

```diff
@@ -0,0 +1,134 @@
+{
+    "app-id": "org.nomacs.ImageLounge",
+    "branch": "3.14",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.14",
+    "sdk": "org.kde.Sdk",
+    "finish-args": [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=home"
+    ],
+    "command": "nomacs",
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
+        "/share/aclocal",
+        "/share/man",
+        "*.la", "*.a"
+    ],
+    "build-options": {
+        "arch": {
+            "x86-64": {
+                "no-debuginfo": true,
+                "strip": true
+            },
+            "aarch64": {
+                "no-debuginfo": false,
+                "strip": false
+            },
+            "arm": {
+                "no-debuginfo": false,
+                "strip": false
+            }
+        }
+    },
+    "modules": [
+        {
+            "name": "exiv2",
+            "buildsystem": "cmake",
```

Comment:
`cmake-ninja`

Reply:
stupid question: should I replace all "cmake" with "cmake-ninja" or just this one?

Reply:
all that work with ninja

Reply:
Yeah, assuming it works it just makes everything faster.